src/App.js

```diff
@@ -1,13 +1,28 @@
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import "./App.css";
-import Home from '../src/Pages/Home';
+
+import Home from "../src/Pages/Home";
+import ArticlePage from "./Components/ArticlePage";
+import Header from "./Header/Header";
+import Sidebar from "./Sidebar/Sidebar";
+import Footer from "./Footer/Footer";
 
 function App() {
   return (
     <Router>
-      <Routes>
-        <Route path="/" element={<Home />} />
-      </Routes>
+      <div className="app__container">
+        <div className="sidebar__container">
```

Comment:
try to keep the use of divs to a minimum and reserve for when there isn't a more appropriate semantic tag available - is this a nav bar for example?